packages/fxa-content-server/app/scripts/views/settings.js

```diff
@@ -292,6 +297,18 @@ const View = BaseView.extend({
     }, this.SUCCESS_MESSAGE_DELAY_MS);
     return BaseView.prototype.unsafeDisplaySuccess.apply(this, arguments);
   },
+
+  showSecurityEvents() {
+    if (this.window.location.href.includes('security_events=true')) {
```

Comment:
While this does work, it is better to set the broker capabilities similar to https://github.com/mozilla/fxa/blob/33342a9dca24818fbed0b8ac1cf296e8d80998c4/packages/fxa-content-server/app/scripts/models/auth_brokers/base.js#L52.